### ISSUE TITLE
Selawik doesn't work with my configuration

### ISSUE THREAD

**Issue by Nevvulo** [selawik/selawik-os#2]
Title says it all. My system doesn't support Selawik. When I clone Selakwik using `git clone`, my computer doesn't boot into anything and simply prints "No operating system loaded, press any key to restart..."

**Comment by reflectronic** [comment]
Have you tried turning it off and on again?

**Comment by ntfluoride** [comment]
#MAGA #TRUMPTRAIN #TRUMP2020

**Comment by vicr123** [comment]
Of course, the only explanation is S#

**Comment by Nevvulo** [comment]
@reflectronic Yeah, unfortunately the issue still occurs. Here's a screenshot of what the error in question looks like: ![image](https://massive-legend.nevulo.xyz/fMIRJgxf.png)
I appreciate your time.

**Comment by reflectronic** [comment]
Yeah the statics were not able to mutate enough to boot in your case. Sorry about that.

**Comment by ntfluoride** [comment]
LIAR FAKE NEWS

**Comment by ntfluoride** [comment]
I CAN'T LIIIIIIIVVVEEEEEEE

**Comment by ntfluoride** [comment]
IF LIVIN' IS WITHOUT YOU

**Comment by reflectronic** [comment]
I think someone forgot an `ignore` keyword somewhere...

**Comment by ntfluoride** [comment]
oops, wrong window

**Comment by vicr123** [comment]
@ntfluoride maybe clam yourself

**Comment by Nevvulo** [comment]
Do you know how I can add mutation to statics without using JavaScript?

**Comment by ntfluoride** [comment]
did you just call me a clam

**Comment by vicr123** [comment]
Have an oyster 🦪

**Comment by ntfluoride** [comment]
🦪?

**Comment by reflectronic** [comment]
https://emojipedia.org/emoji/%F0%9F%A6%AA/

**Comment by ntfluoride** [comment]
Hello, I tried running your product and it did this to my president...

![trumpian](https://user-images.githubusercontent.com/13461827/52549719-6896e600-2d89-11e9-97bb-5e8dbf03193a.png)
